docs/scos/dev/tutorials-and-howtos/howtos/howto-import-big-databases-between-environments.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "HowTo: Import big databases between environments"
+description: Learn how you can relatively quickly import big data between your environments
+last_updated: April 5, 2023
+template: howto-guide-template
+---
+
+Suppose you have two testing environments, and you need to migrate a large amount of data from one environment to another to perform different tests with the same data. If you have little data, you can export it using the `mysqldump` command. However, for large amounts of data, this method can be slow due to long waiting time and VPN connection issues. In this case, you can use AWS S3 to efficiently import the data between the environments. Here's how to do it:
```

Comment:
We use `mysqldump` command for both large and small data. The difference here is where we run this command.

If the data set is small, then we can run this command on our local machine.
If the data set is large, then we have to run this command on Jenkins instance, in AWS, and upload it to S3.

Reply:
fixed this line